src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1202,7 +1202,9 @@ namespace ZoneEquipmentManager {
                 CalcZoneSizing(DesDayNum, CtrlZoneNum).HeatLoadSeq.allocate(NumOfTimeStepInDay);
                 CalcZoneSizing(DesDayNum, CtrlZoneNum).CoolLoadSeq.allocate(NumOfTimeStepInDay);
                 CalcZoneSizing(DesDayNum, CtrlZoneNum).HeatZoneTempSeq.allocate(NumOfTimeStepInDay);
+                CalcZoneSizing(DesDayNum, CtrlZoneNum).DesHeatSetPtSeq.allocate(NumOfTimeStepInDay);
```

Comment:
New method required uniformity in allocation of arrays for ZoneSizing, CalcZoneSizing, FinalZoneSizing and CalcFinalZoneSizing.

Reply:
Seems like another method could be used here to allocate all arrays.